services/processor/native/sanitizer/test/config.go

```diff
@@ -13,6 +13,7 @@ func SanitizerConfigForTests() *sanitizer.SanitizerConfig {
 		ImportWhitelist: map[string]bool{
 			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1"`:       true,
 			`"github.com/orbs-network/orbs-contract-sdk/go/sdk/v1/state"`: true,
+			`"time"`: true,
```

Comment:
why? do we want to allow contracts to sleep?

Reply:
No, but they might want to do date conversion and date-based checks.